setup.py

```diff
@@ -27,10 +30,11 @@
 
 setup(
     name="cfripper",
-    version="0.12.0",
+    version=__version__,
     author="Skyscanner Product Security",
     author_email="security@skyscanner.net",
-    long_description=long_description,
+    entry_points={"console_scripts": ["cfripper=cfripper.cli:cli"]},
+    long_description=(project_root_path / "README.md").read_text(),
     long_description_content_type="text/markdown",
     url="https://github.com/Skyscanner/cfripper",
     description='Lambda function to "rip apart" a CloudFormation template and check it for security compliance.',
```

Comment:
this is no longer a correct description with this patch

Reply:
Thanks! I will update description